connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/model/HttpServletRequestWebhookProcessingPayload.java

```diff
@@ -35,12 +36,18 @@ public HttpServletRequestWebhookProcessingPayload(
       final Map<String, String> params,
       final Map<String, String> headers,
       byte[] bodyAsByteArray) {
+    this.requestURL = httpServletRequest.getRequestURL().toString();
```

Comment:
👍

Reply:
We could also just get the whole URL including parameters and put into a type `URL`. Then we dont have to add the parameters again afterwards.